tasks-by-users/mmusk/02-second-task-css/product details.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>All Products - RedStore</title>
+    <link rel="stylesheet" href="style.css" />
```

Comment:
custom style should come last.
how will they know if you wanna use font stylle from google fonts, but they are loaded after your css styles ?

Reply:
fixed @faran4engg